utilities/src/main/java/io/syndesis/qe/templates/SftpTemplate.java

```diff
@@ -0,0 +1,111 @@
+package io.syndesis.qe.templates;
+
+import io.syndesis.qe.TestConfiguration;
+import io.syndesis.qe.utils.OpenShiftUtils;
+import io.syndesis.qe.utils.TestUtils;
+import io.syndesis.qe.wait.OpenShiftWaitUtils;
+
+import java.util.ArrayList;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.concurrent.TimeoutException;
+
+import io.fabric8.kubernetes.api.model.ContainerPort;
+import io.fabric8.kubernetes.api.model.ContainerPortBuilder;
+import io.fabric8.kubernetes.api.model.EnvVar;
+import io.fabric8.kubernetes.api.model.IntOrString;
+import io.fabric8.kubernetes.api.model.ServicePortBuilder;
+import io.fabric8.kubernetes.api.model.ServiceSpecBuilder;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class SftpTemplate {
+    public static final int SFTP_PORT = 22;
+    private static final String APP_NAME = "sftpd";
+    private static final String LABEL_NAME = "app";
+    public static final String TEST_DIRECTORY = "test";
+    private static final String USER_AND_PASSWORD = "syndesis-qe:syndesis2020:1500::" + TEST_DIRECTORY;
+
+    public static void deploy() {
+        //preparation for our specific SFTP image to fit Openshift requirements:
+        OpenShiftUtils.binary().execute("create", "serviceaccount", "mysvcacct", "-n", TestConfiguration.openShiftNamespace());
+        OpenShiftUtils.binary().execute("adm", "policy", "add-scc-to-group", "anyuid", "system:authenticated");
+        OpenShiftUtils.binary()
+            .execute("adm", "policy", "add-scc-to-user", "anyuid", "system:serviceaccount:" + TestConfiguration.openShiftNamespace() + ":mysvcacct");
```

Comment:
please avoid using binary client if not necessary, these commands should be possible to do with kubernetes client